src/components/AgmExtension/ExtensionRequest.vue

```diff
@@ -126,21 +163,40 @@
 </template>
 
 <script lang="ts">
-import { Component, Emit, Prop, Vue } from 'vue-property-decorator'
+import { Component, Emit, Prop, Vue, Watch } from 'vue-property-decorator'
+import { Getter } from 'pinia-class'
 import { VcardTemplate } from '@/components/common'
 import { AgmExtEvalIF } from '@/interfaces'
+import { useRootStore } from '@/stores'
 import { DatePicker } from '@bcrs-shared-components/date-picker'
+import AgmYear from '@/components/AgmLocationChange/AgmYear.vue'
+import { DateUtilities } from '@/services'
 
 @Component({
   components: {
+    AgmYear,
     DatePicker,
     VcardTemplate
   }
 })
 export default class ExtensionRequest extends Vue {
   @Prop({ required: true }) readonly data!: AgmExtEvalIF
+
   @Prop({ default: false }) readonly showErrors!: boolean
 
+  @Getter(useRootStore) getCurrentDate!: string
+
+  // Has extension been requested date text
+  extensionExpiryDateText = ''
```

Comment:
1. If you use JSDoc comment style, hovering over the variable shows your comments.
2. This isn't a boolean so the comment needs to be updated.

Reply:
Thanks. Looks good.